test/test_checker.ts

```diff
@@ -735,6 +735,21 @@ function suite() {
       {"path": "value.c", "message": "is missing"},
     );
   });
+
+  it("should name array types in error messages", () => {
+    const {MixedArray, ArrayUnion} = createCheckers(enumUnionTI);
+
+    MixedArray.check(['one', 'two', 'three']);
+    MixedArray.check(['one', 'two', 'three']);
+    MixedArray.check([1, 'two', 3]);
+    assert.throws(() => MixedArray.check([undefined]), "value[0] is none of string, number, string[]");
+
+    ArrayUnion.check(['one', 'two', 'three']);
+    ArrayUnion.check([1, 2, 3]);
+    ArrayUnion.check([true, false]);
+    assert.throws(() => ArrayUnion.check([undefined]), "value is none of string[], boolean[], number[], string[][]");
```

Comment:
Please add a test that includes an array with an unnamed type.

Reply:
Updated